esm-samples/rollup/README.md

```diff
@@ -8,4 +8,24 @@ Be sure to set [`config.assetsPath`](https://developers.arcgis.com/javascript/la
 import esriConfig from '@arcgis/core/config.js';
 
 esriConfig.assetsPath = './assets'; 
+```
+
+The `rollup.config.js` file handles the copying of the API's assets for watch and build.
+
+```js
+
+import copy from 'rollup-plugin-copy';
+
+plugins: [
+  ...
+    copy({
+      // Copy the ArcGIS APi for JavaScript assets
+      targets: [
+        { src: './node_modules/@arcgis/core/assets/*', dest: './public/assets'},
```

Comment:
Update to match usage

Reply:
Good catch. Now I'm wondering with the assets being copied via `rollup -c` if the postinstall script is redundant. Thoughts?

```js
  "scripts": {
    "build": "rollup -c",
    // "copy": "ncp ./node_modules/@arcgis/core/assets ./public/assets",
    "watch": "rollup -c -w",
    "dev": "npm-run-all --parallel start watch",
    "start": "serve public",
    // "postinstall": "npm run copy"
  }
```

Reply:
Yes, that can be removed.

Reply:
Removed.